hraven-etl/src/main/java/com/twitter/hraven/mapreduce/GraphiteOutputFormat.java

```diff
@@ -163,6 +167,50 @@ public void write(EnumWritable<HravenService> serviceKey, HravenRecord value) th
           throw new IOException(e);
         }  
       }
+        } else {
+            String multipleConfigInput = context.getConfiguration().get("hraven.multiple.config.files");
+            String[] multipleConfigs = multipleConfigInput.split(",");
+            for (String config : multipleConfigs) {
+                String configStr = readFsFile(config, new Configuration());
+                HashMap<String,String> configMap = new Gson().fromJson(configStr.toString(), new TypeToken<HashMap<String, String>>(){}.getType());
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_PREFIX))
+                    METRIC_PREFIX = configMap.get(Constants.JOBCONF_GRAPHITE_PREFIX);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_USER_FILTER))
+                    userFilter = configMap.get(Constants.JOBCONF_GRAPHITE_USER_FILTER);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_QUEUE_FILTER))
+                    queueFilter = configMap.get(Constants.JOBCONF_GRAPHITE_QUEUE_FILTER);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_EXCLUDED_COMPONENTS))
+                    excludedComponents = configMap.get(Constants.JOBCONF_GRAPHITE_EXCLUDED_COMPONENTS);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_INCLUDE_APPS))
+                    appInclusionFilter = configMap.get(Constants.JOBCONF_GRAPHITE_INCLUDE_APPS);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_EXCLUDE_APPS))
+                    appExclusionFilter = configMap.get(Constants.JOBCONF_GRAPHITE_EXCLUDE_APPS);
+                if(configMap.containsKey(Constants.JOBCONF_GRAPHITE_NAMING_RULE_CONFIG))
+                    metricNamingRules = configMap.get(Constants.JOBCONF_GRAPHITE_NAMING_RULE_CONFIG);
+
+                try {
+                    GraphiteHistoryWriter graphiteWriter =
+                        new GraphiteHistoryWriter(context, keyMappingTable, reverseKeyMappingTable, METRIC_PREFIX, service, recordCollection, output,
```

Comment:
the hadoop counters now need to be prefixed with the config "name" (can be derived from config file name)